sycl/include/sycl/detail/pi.h

```diff
@@ -1033,6 +1039,14 @@ __SYCL_EXPORT pi_result piDevicesGet(pi_platform platform,
                                      pi_uint32 num_entries, pi_device *devices,
                                      pi_uint32 *num_devices);
 
+__SYCL_EXPORT pi_result piextEnablePeer(pi_device command_device,
+                                        pi_device peer_device);
+__SYCL_EXPORT pi_result piextDisablePeer(pi_device command_device,
+                                         pi_device peer_device);
+__SYCL_EXPORT pi_result piextCanAccessPeer(pi_device command_device,
+                                           pi_device peer_device,
+                                           pi_peer_attr attr);
+
```

Comment:
New interfaces mean `_PI_H_VERSION_MINOR` should be incremented.

Reply:
ta